src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,96 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (event: React.FormEvent<HTMLAnchorElement>) => void;
+  onPerPageChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total, perPage, currentPage, onPageChange, onPerPageChange,
+}) => {
+  const arrOfPages = Array.from({ length: total }, (_, i) => i + 1);
+
+  return (
+    <>
+      <div className="form-group row">
+        <div className="col-3 col-sm-2 col-xl-1">
+          <select
+            data-cy="perPageSelector"
+            id="perPageSelector"
+            className="form-control"
+            defaultValue={perPage}
+            onChange={(event) => onPerPageChange(event)}
+          >
+            <option value="3">3</option>
+            <option value="5">5</option>
+            <option value="10">10</option>
+            <option value="20">20</option>
+          </select>
+        </div>
```

Comment:
Pagination el dont need to have select, its only the nav
https://getbootstrap.com/docs/4.0/components/pagination/